Cargo.toml

```diff
@@ -51,7 +52,7 @@ tokio = { version = "1.35.1", default-features = false }
 tracing = "0.1.40"
 tracing-subscriber = "0.3.18"
 up-rust = "0.1.5"
-zenoh = { version = "0.11.0", features = ["unstable"]}
+zenoh = { version = "1.0.0-alpha.6", features = ["unstable", "internal"]}
```

Comment:
This is because runtime is moved under internal features in Zenoh 1.0
https://github.com/eclipse-zenoh/zenoh/blob/0bc608836e20c3ab5ad2e479e8b1aa53ba3addd4/zenoh/src/lib.rs#L403

Reply:
Are there any differences in stability guarantees due to it being being the internal feature flag?

What's the intention behind it? If it's described somewhere, could you post a link?

Reply:
As far as I know, Zenoh 1.0 will try its best to ensure API backward compatibility, but for something related to the plugin (Runtime), the API might be changed in the future. That's why it moved to internal.